README.md

```diff
@@ -101,6 +102,13 @@ This will install the aicspylibczi python module and extension binaries ([hosted
 pip install aicspylibczi
 `
 
+**If this doesn't work:** Please investigate the following (generally windows issues):
+* your OS is 64 bit - we only support 64 bit binaries
+* your python is a 64 bit application (not 32 bit)
+* are your C++ runtime libraries up to date? [vc_redist.exe](https://aka.ms/vs/16/release/vc_redist.x64.exe)
```

Comment:
Might be better to link to `https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads` instead and tell users to download and install the latest x64 version.  I suppose either way, the link might go stale...

Reply:
I was debating that, I might put both in so there's some fallback but yeah it is a little fragile.